README.adoc

```diff
@@ -24,6 +24,11 @@ toc::[]
 [[img-build-status]]
 image::https://ci.centos.org/buildStatus/icon?job=vagrant-service-manager[link="https://ci.centos.org/job/vagrant-service-manager"]
 
+== Prerequisites
```

Comment:
@hferentschik the main reason for adding this section is to allow users to jump "upwards" towards the main ADB install instructions and review any limitations or OS-specific prerequisites. In the PR description you'll see there's a related PR in the ADB repo for this.